common/txmgr/types/storage_service.go

```diff
@@ -6,30 +6,33 @@ import (
 
 	uuid "github.com/satori/go.uuid"
 
+	"github.com/smartcontractkit/chainlink/v2/common/types"
 	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
 )
 
 // NEWTX, TX, TXATTEMPT will be converted from generic types to structs at a future date to enforce design and type checks
-type TxStorageService[ADDR any, CHAINID any, HASH any, NEWTX any, R any, TX any, TXATTEMPT any, TXID any, TXMETA any] interface {
+//
+//go:generate mockery --quiet --name TxStorageService --output ./mocks/ --case=underscore
+type TxStorageService[ADDR any, CHAINID any, TX_HASH types.Hashable, BLOCK_HASH types.Hashable, NEWTX any, R any, TX any, TXATTEMPT any, TXID any, TXMETA any] interface {
```

Comment:
Maybe a case for one type per line?
```suggestion
type TxStorageService[
	ADDR any, 
	CHAINID any, 
	TX_HASH types.Hashable, 
	BLOCK_HASH types.Hashable, 
	NEWTX any, 
	R any, // Receipt?
	TX any, 
	TXATTEMPT any, 
	TXID any, 
	TXMETA any,
] interface {
```

Reply:
thanks, done